### ISSUE TITLE
Integrate CAN Wrapper with GUI

### ISSUE THREAD

**Issue by AndreasFreund** [amosproj/amos2024ss07-updating-flash-boot-loader#19]
## User story
1. As a user
2. I want to send a simple CAN message when pressing a button
3. So that integration between CAN Wrapper and GUI is demonstrated

## Acceptance criteria
* When pressing a button in the GUI, a predefined CAN-Mesage is sent out
* Build process not overly complicated

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
